PortMaster/pylibs/harbourmaster/config.py

```diff
@@ -94,11 +94,11 @@
     HM_DEFAULT_PORTS_DIR   = Path("/storage/roms/ports")
     HM_DEFAULT_SCRIPTS_DIR = Path("/storage/roms/ports")
 
-## Check if retrodeck.cfg exists. Chose this file/location as platform independent from were retrodeck is installed.
-elif (Path.home() / ".var/app/net.retrodeck.retrodeck/config/retrodeck/retrodeck.cfg").is_file():
-    retrodeck_roms_path = retrodeck_roms_path = (Path.home() / ".var/app/net.retrodeck.retrodeck/config/retrodeck/retrodeck.cfg").read_text()
+## Check if retrodeck.sh exists. Chose this file/location as platform independent from were retrodeck is installed.
+elif ("/var/app/bin/retrodeck.sh").is_file():
+    retrodeck_roms_path = retrodeck_roms_path = ("/var/config/retrodeck/retrodeck.cfg").read_text() # is it possible to read from the bash variable $roms_dir and elaborarte that as done in the .sh files?
```

Comment:
Same here, also double assignment?
```suggestion
    retrodeck_roms_path = Path("/var/config/retrodeck/retrodeck.cfg").read_text() # is it possible to read from the bash variable $roms_dir and elaborarte that as done in the .sh files?
```